scope/src/analyze/cli.rs

```diff
@@ -28,9 +37,20 @@ struct AnalyzeLogsArgs {
     location: String,
 }
 
+#[derive(Debug, Args)]
+struct AnalyzeCommandArgs {
+    /// The command to run
+    #[arg(required = true)]
+    command: String,
```

Comment:
I think we'll want to use https://docs.rs/clap/latest/clap/_derive/_cookbook/escaped_positional_derive/index.html to take in the command, that way clap won't consume the global args on accident.

Reply:
Great catch!